core/catalog-core-api-impl/src/main/java/ddf/catalog/data/impl/BasicTypes.java

```diff
@@ -306,6 +306,9 @@ public AttributeFormat getAttributeFormat() {
                 true /* stored */, false /* tokenized */, false /* multivalued */, XML_TYPE));
         descriptors.add(new AttributeDescriptorImpl(Metacard.RESOURCE_URI, true /* indexed */,
                 true /* stored */, false /* tokenized */, false /* multivalued */, STRING_TYPE));
+        descriptors.add(new AttributeDescriptorImpl(Metacard.RESOURCE_DOWNLOAD_URL,
+                true /* indexed */, true /* stored */, false /* tokenized */,
+                false /* multivalued */, STRING_TYPE));
```

Comment:
Good catch. This was added by mistake in a previous commit. Undoing the change.

Regarding your other point, we've debated this quite a bit and came to the conclusion that it was simpler to always include it in the metacard on the way out. That way, all endpoints will always return it and clients will not have to explicitly convert the resource-uri or have to deal use actions.

This could also have been achieved using a post-query plugin but we felt that there wasn't a good use case to create a plug-in that would always be installed by default and shouldn't be removed.

Hope this helps. @pklinef and @kcwire can provide more information on this if you'd like.